connector/mqtt/vernemq/broker/src/dojot_acl_plugin/src/dojot_acl_plugin.erl

```diff
@@ -0,0 +1,47 @@
+-module(dojot_acl_plugin).
+
+-behaviour(auth_on_register_hook).
+-behaviour(auth_on_subscribe_hook).
+-behaviour(auth_on_publish_hook).
+
+-export([auth_on_register/5,
+         auth_on_publish/6,
+         auth_on_subscribe/3]).
+
+auth_on_register({_IpAddr, _Port} = Peer, {_MountPoint, _ClientId} = SubscriberId, UserName, Password, CleanSession) ->
+    ok.
+
+auth_on_publish(UserName, {_MountPoint, _ClientId} = SubscriberId, QoS, Topic, Payload, IsRetain) ->
+    Result  = dojot_acl:check_valid_topic(UserName, Topic),
+    Chained = erlang:list_to_binary([os:getenv("PLUGIN_ACL_CHAIN")]),
```

Comment:
There is no default value set, which causes an error when not passing this variable.